### ISSUE TITLE
State of the Emulator

### ISSUE THREAD

**Issue by Disctanger** [Jonathan57500/aion-lightning-2.7#2]
Good day @Jonathan57500 
How complete is this emulator?
Is there list of known bugs?

**Comment by Jonathan57500** [comment]
Hi @Disctanger 
First, thank you for your interest to this project.
This emulator is working, if you want a %age of advancement I would say something like 80 / 85%. With this emulator, you can play Aion like it was in 2011 when it starts Free-to-Play model in Europe.
In the Wiki you can find some info about what is new in this emulator but I don't want to add my own known bug list, I would like other developers to participate on it. To give you an overview, you'll have some skill issues (but PvP and RvR is largely playable), some missing quests (for the most, useless quests), some non-existent features like Dreadgion Strikes (there is some code but it never worked) or low level PvP curses, unachieved arena and coliseum.
But don't worry, this emulator was used in 2 different servers with different gameplay orientation last 4 years and globally players said it was a correct emulator, not perfect but largely playable.

**Comment by Disctanger** [comment]
Thanks for quick reply, @Jonathan57500 
That is interesting to know. Long time ago I used to work with emulator called _JS_. I remember at that time none of the 2.7 was complete and a lot of stuffs (especially instances such as Padmarashka and Reshanta) used not to work properly. _JS_ used to take a huge amount of time to fix and maintain the source code eventually I gave up server due to time prolems. But I understand it is impossible to make any kind of emulator as same as official version. Now I am considering whether to deal with Aion again or not. If I happen to work with Aion, I will surely join development and help as much as possible.

**Comment by Jonathan57500** [comment]
That's a good news ;)
Yes, Padmarashka is not yet implemented because it's hardest instance in 2.7 and because players have to do it as big group (can't remember if it's 24 or 48 players max) it's not useful to make it work great as players must wait a server to get a certain amount of lvl max players. Loots are not setted up on this boss.
I'm very happy to give you some help if you want to make your own server. First step is to identify all customs that are currently installed in this emulator, customs used on other severs, and change them by a better offlike value.

**Comment by Jonathan57500** [comment]
Any news here? :)

**Comment by Disctanger** [comment]
Not yet, I did not check the instances yet. I am still level 7 :sweat_smile:

**Comment by Jonathan57500** [comment]
You know, in your server you can set your account as access level 12 and become an administrator, then update your character from 7 to 55 in 5 seconds :)

**Comment by Disctanger** [comment]
Yeah I know about //set command but just taking my time and trying to review the emulator step by step
